src/data/finset/interval.lean

```diff
@@ -87,4 +87,12 @@ by rw [card_Ioc_eq_card_Icc_sub_one, card_Icc_finset h]
 lemma card_Ioo_finset (h : s ⊆ t) : (Ioo s t).card = 2 ^ (t.card - s.card) - 2 :=
 by rw [card_Ioo_eq_card_Icc_sub_two, card_Icc_finset h]
 
+/-- Cardinality of an `Iic` of finsets. -/
+lemma card_Iic_finset : (Iic s).card = 2 ^ s.card :=
```

Comment:
I think we could make `Iic_eq_powerset` simp instead of proving this lemma.

Reply:
That would mean that lemmas like `Iic_union_Ioc` or whatever the name is or could be wouldn't apply. I'd lean towards playing it safe here for now and having both lemmas, as that's also better for discoverability.

Reply:
But then `powerset` and `Iic` are two spellings for the exact same object. Maybe we could go as far as not defining `finset.powerset` anymore?

Reply:
I don't think this is meaningfully different from having `inter` vs `inf` or `subset` vs `le`; `powerset` is the set spelling, `finset.Iic` is the lattice spelling.

Reply:
But in that case we swear to never use `inf` and `sup` on `set`/`finset`. And here you're not willing to swear to never use `finset.Iic` on `finset`.

Reply:
> And here you're not willing to swear to never use finset.Iic on finset.

Swearing that seems out of scope for this PR. Would you rather the PR go ahead as is, or that I revert the changes to this file? I'm only really interested in the multiset results in this PR, and would rather not hold them up on finset `simp`-normal form.

Reply:
We can leave it like that for now.